pkg/types/baremetal/validation/platform_test.go.orig

```diff
@@ -0,0 +1,721 @@
+package validation
```

Comment:
I think this file was added by mistake? :)

Reply:
Yep, I had some trouble with `patch` which led to adding some extra files, fixed now, thanks!